### PR TITLE
GUACAMOLE-35: Ensure performance flag values are applied.

### PR DESCRIPTION
**Pull request by mike-jumper**

### CODE REVIEW

File: src/protocols/rdp/rdp_settings.c [hunk 1]
```diff
@@ -925,9 +925,31 @@ void guac_rdp_push_settings(guac_rdp_settings* guac_settings, freerdp* rdp) {
...
+    /* Explicitly set flag value */
     rdp_settings->performance_flags = guac_rdp_get_performance_flags(guac_settings);
+
+    /* Set individual flags - some FreeRDP versions overwrite the above */
+    rdp_settings->smooth_fonts = guac_settings->font_smoothing_enabled;
+    rdp_settings->disable_wallpaper = !guac_settings->wallpaper_enabled;
+    rdp_settings->disable_full_window_drag = !guac_settings->full_window_drag_enabled;
+    rdp_settings->disable_menu_animations = !guac_settings->menu_animations_enabled;
+    rdp_settings->disable_theming = !guac_settings->theming_enabled;
+    rdp_settings->desktop_composition = !guac_settings->desktop_composition_enabled;
```
Comment (jmuehlner) [inline src/protocols/rdp/rdp_settings.c]: Should this be inverted here?

Comment (mike-jumper) [inline src/protocols/rdp/rdp_settings.c]: Yes.

Comment (jmuehlner) [inline src/protocols/rdp/rdp_settings.c]: Meaning that you should remove the ! that inverts the setting here?

Comment (mike-jumper) [inline src/protocols/rdp/rdp_settings.c]: Yes.

**Pull request merged**